lib/rules/ngx-component-display.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
       recommended: 'error',
     },
     messages: {
-      missingProperty: `Missing '{{ propertyName }}' property in component '{{ name }}'. Please define '@HostBinding('class.cds-display-block') readonly {{ propertyName }} = true;'. Valid values: cds-display-block, cds-display-inline, cds-display-inline-block`,
+      missingProperty: `Missing '{{ propertyName }}' property in component '{{ name }}'. Please define '@HostBinding('class.cds-display-block') readonly {{ propertyName }} = true;'. Valid values: cds-display-block, cds-display-inline, cds-display-inline-block. \n\nIf your component is a dialog or a modal, ignore this rule and rename your component to end with 'DialogComponent' or 'ModalComponent'`,
```

Comment:
This is customizable, so not necessary true. You can say "With the default config, this rule is disabled for components whose names end with 'DialogComponent' or 'ModalComponent'."

Do not insert blank lines with "\n\n" otherwise the textual result of the ESlint command line is hard to understand.

Reply:
Will change it! 

I have founded the "\n\n"` better in term of visibility but let's try without